### PR TITLE
Add data conversions for extra BrAPI data types

### PR DESCRIPTION
**Pull request by ctucker3**
Added conversions for data types that were left out of the original conversion. Converted the data types to the closest matches in Fieldbook:

"Code" - This scale class is exceptionally used to express complex traits. Code is a nominal scale that combines the expressions of the different traits composing the complex trait. For exemple a severity trait might be expressed by a 2 digit and 2 character code. The first 2 digits are the percentage of the plant covered by a fungus and the 2 characters refer to the delay in development, e.g. "75VD" means "75%" of the plant is Crop Ontology & Integrated Breeding Platform Curation Guidelines 5/6/2016 9 infected and the plant is very delayed.
"Date" - The date class is for events expressed in a time format, e.g. yyyymmddThh:mm:ssZ or dd/mm/yy
"Duration" - The Duration class is for time elapsed between two events expressed in a time format, e.g. days, hours, months
"Nominal" - Categorical scale that can take one of a limited and fixed number of categories. There is no intrinsic ordering to the categories
"Numerical" - Numerical scales express the trait with real numbers. The numerical scale defines the unit e.g. centimeter, ton per hectar, branches
"Ordinal" - Ordinal scales are scales composed of ordered categories
"Text" - A free text is used to express the trait.

Conversions are: 

Code -> text
Ordinal -> categorical
Numerical -> numeric

### CODE REVIEW

**Comment by ctucker3** [comment]
[BBase-8 - GET variables - map breedbase datatypes to BrAPI datatypes](https://trello.com/c/8XGWBvbk/61-bbase-8-get-variables-map-breedbase-datatypes-to-brapi-datatypes)

**Comment by trife** [comment]
The Code data type sounds similar to the current disease rating trait. It might be worth replacing the disease rating trait code with something that can be used more generally for Code data types if that is a popular format among breeders.

**Comment by ctucker3** [comment]
Good thinking! I haven't looked too much into the disease rating trait, but I think I am following your train of thought. I think for now, we can leave that outside of this release and make that a future effort. Thoughts on that?

File: app/src/main/java/com/fieldbook/tracker/brapi/BrAPIService.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public BrAPIService(String brapiBaseURL, DataHelper dataHelper) {
 
         ApiClient apiClient = new ApiClient().setBasePath(brapiBaseURL);
 
-        // Make timeout longer. Set it to 60 seconds for now
+        // Make timeout longer. Set it to 120 seconds for now
```
[the remark this answers is not in the document]
Comment (ctucker3) [reply app/src/main/java/com/fieldbook/tracker/brapi/BrAPIService.java]: Thanks for the catch! Change it in testing and then switched the time back, but not the comment. I just fixed it.

**Comment by trife** [comment]
Yeah totally agree that it should be a future effort. Just getting the idea out there.

**Pull request merged**